js/main.js

```diff
@@ -12,4 +12,11 @@ const options = {
 };
 
 const game = new Grid(options.gridSize.width, options.gridSize.height, new HTMLRenderer('#snake-game'));
-game.start();
+
+var startButton = document.getElementById('start');
+var resetButton = document.getElementById('reset');
```

Comment:
Gotta finish this. I'll add a 'reset' method to the Grid object that we can call directly from here so we can keep `main.js` as the "glue" for things, rather than letting game logic leak out here.